nni/experiment/launcher.py

```diff
@@ -129,12 +149,12 @@ def start_experiment(action, exp_id, config, port, debug, run_mode, url_prefix):
 
     return proc
 
-def _start_rest_server(nni_manager_args, run_mode) -> Popen:
+def _start_rest_server(nni_manager_args: NniManagerArgs, run_mode: RunMode) -> Popen:
     import nni_node
     node_dir = Path(nni_node.__path__[0])  # type: ignore
     node = str(node_dir / ('node.exe' if sys.platform == 'win32' else 'node'))
     main_js = str(node_dir / 'main.js')
-    cmd = [node, '--max-old-space-size=4096', main_js]
+    cmd = [node, '--max-old-space-size=4096', '--trace-uncaught', main_js]
```

Comment:
what is `--trace-uncaught` used for?

Reply:
Print stack trace when an uncaught exception is thrown in async function. For better debugability.